docs/source/config.rst

```diff
@@ -351,22 +351,18 @@ FMS based model only options:
    is then ``ncpus / nthreads``
 
 
-Postprocessing
+User Processing
 --------------
 
-``collate`` (*Default:* ``True``)
-   Controls whether or not a collation job is submitted after model execution.
-
-   This is typically ``True``, although individual model drivers will often set
-   the default value to ``False`` if collation is unnecessary.
-
-   See above for specific ``collate`` options.
-
 ``userscripts``
-   Namelist to include separate userscripts or subcommands at various stages of
+   Configure userscripts or subcommands to run at various stages of
    a payu submission. Inputs can be either script names (``some_script.sh``) or
    individual subcommands (``echo "some_data" > input.nml``, ``qsub
-   some_script.sh``).
+   some_script.sh``). Userscripts are run within the same PBS job as the model 
+   execution unless the script starts a new PBS job. Userscripts therefore have
+   the same compute, storage and network access as the model. The exception to 
+   this is when `payu setup` is called directly, then the relevant userscripts 
+   will run on the login node.
```

Comment:
Another exception is the `sync` user scripts which runs on the `sync` pbs job. It's noted under the `sync` user script sub-section. Maybe the sync userscript section should be extended it make it a little more clear? e.g.:

""User-defined command to be called at the start of the `sync` PBS job. This is useful for any post-processing before syncing files to a remote archive. Note these scripts are only run within `sync`  PBS job so will not run unless automatic syncing is enabled or if `payu sync` is run manually.""